### ISSUE TITLE
several issues

### ISSUE THREAD

**Issue by CeresBarros** [CeresBarros/SpaDES4Dummies#10]
- Change `bind_rows` to `bindrows`
- `speciesTempLM` module is labelled `speciesAbundance` in the module code, so `SpaDES.core` balks at `simInit`
- The inputs and outputs in the metadata do not match the `moduleDiagram `figure ... `r` is the only input for everything ... the figure shows that is not the case.
- This function needs `Time` to pass into `abundance_mode`l which it doesn't provide

```
abundanceSim <- function(sim) {
  ## Generate species abundances - our "simulation"
  sim$abundRasters[[as.character(time(sim))]] <- abundance_model(ras = sim$r)

  return(invisible(sim))
}
```

**Comment by CeresBarros** [comment]
this:

> This function needs Time to pass into abundance_model which it doesn't provide
> abundanceSim <- function(sim) {
>   ## Generate species abundances - our "simulation"
>   sim$abundRasters[[as.character(time(sim))]] <- abundance_model(ras = sim$r)
> 
>   return(invisible(sim))
> }

was wrong, as it referred to the functon used  in the "tradition R way"example, not the module function.
The environment is now cleaned before running SpaDES modules in the Rmd knit (e4fe07bfea45b7651b8779dbeaa09494d77f0901)

**Comment by CeresBarros** [comment]
remaining issues solved in: 8e5b6267a773b307daeda982362b30eeb24434a3
